framework/include/kernels/MaterialDerivativeTestKernel.h

```diff
@@ -0,0 +1,54 @@
+/****************************************************************/
```

Comment:
Yes, we want to be able to to use it in other apps. Apps do not link objects from `test`.